MAGECLOUD-3913: Deadlocks when consumers run from cron (#5141)
magento/devdocs: guides/v2.2/config-guide/mq/manage-message-queues.md

*   `--batch-size=<value>` specifies the number of messages to consume per batch. If specified, messages in a queue are consumed in batches of `<value>` each. This option is applicable for the batch consumer only. If `--batch-size` is not defined, the batch consumer receives all available messages in a queue.

*   `--pid-file-path=<value>` the file path for saving PID of consumer process. This option is deprecated, use `--single-thread` instead.

> *   `--single-thread` this option prevents running multiple copies of one consumer simultaneously.

Reviewer: ```suggestion
*   `--single-thread` this option prevents multiple copies of a consumer from running simultaneously.
```

*   `--area-code=<value>` the area code preferred for consumer process.

> {: .bs-callout .bs-callout-warning }

Reviewer: ```suggestion
{: .bs-callout-warning }
```

Reviewer: CSS was updated, so ```.bs-callout``` class is no longer required.

> The option `--pid-file-path` was marked as deprecated in releases 2.2.10 and 2.3.3 and pid files won't be generated anymore.

Reviewer: ```suggestion
The `--pid-file-path` option is deprecated in releases 2.2.10 and 2.3.3. Magento will no longer generate pid files.
```
